src/analyticsTracking/eventUtils.js

```diff
@@ -0,0 +1,119 @@
+import analyticsTrackingConstants from '../common/analyticsTrackingConstants';
```

Comment:
they are used in subsequent PRs, it was easier to include them in this PR to attempt to avoid a long PR dependency chain. This way other PRs can simply be rebased on this one